.circleci/config.yml

```diff
@@ -378,7 +378,7 @@ workflows:
       - centos:
           <<: *on-any-branch
       - macos:
-          <<: *on-version-tags
+          <<: *on-any-branch
```

Comment:
Shouldn't build macOS on any branch. Maybe on nightly? On integ branches?

Reply:
This was to test. But the question generally @oshadmi and I had was why not? Why do we limit what builds where?

Reply:
`on-any-branch` means you'll build for macOS on any push to a PR. This is annoying and wasteful. macOS can be enabled explicitly when needed (for PRs) and we can build them on integ branches (this, however, can also be annoying and wasteful). I think that it is best to do periodic macOS builds (on all live integ branches) just to keep the platform alive.

Reply:
I get that it _could_ be wasteful, but my question is generally why don't we build everything always. On AI we also limited it, and then randomly regret it. But yes, it's restored, resolving.